<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>371</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="coupang_id">
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="coupang_id_edit">
        <property name="placeholderText">
         <string>ID</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="coupang_pw">
        <property name="text">
         <string>PW</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="coupang_pw_edit">
        <property name="inputMask">
         <string/>
        </property>
        <property name="placeholderText">
         <string>password</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="search_word">
        <property name="text">
         <string>검색어</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="search_word_edit">
        <property name="placeholderText">
         <string>예) 물</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="search_count">
        <property name="text">
         <string>개수</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="search_count_edit">
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string>10</string>
        </property>
        <property name="placeholderText">
         <string>1보다 커야합니다</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>카테고리</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="categories"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,0">
      <item>
       <widget class="QPushButton" name="set_dir">
        <property name="text">
         <string>결과 저장 경로 설정</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="dir_edit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>경로를 설정해주세요</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="coupang">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>쿠팡 파트너스 크롤러</string>
   </property>
  </widget>
  <widget class="QPushButton" name="start">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>290</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
